WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/MainState.kt

```diff
@@ -169,7 +169,7 @@ class MainState(
 /**
  * The three playback states of the application.
  */
-sealed class PlaybackState {
+public sealed class PlaybackState {
```

Comment:
is this required? Is this for https://github.com/Kotlin/KEEP/blob/master/proposals/explicit-api-mode.md ?

Reply:
Added to be able to create the PlaybackState from the Screenshot Test

Reply:
Isn't it public by default?

Reply:
you are right, removed as redundant